EnglishFilter/sections/antiadblock.txt

```diff
@@ -568,6 +568,10 @@ uaclips.com,kzclip.com,ru-clip.com,ruclip.com###morev
 !
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/77644
+lightnovelpub.com#%#//scriptlet("prevent-addEventListener", "DOMContentLoaded", "offsetHeight")
+!+ PLATFORM(ios, ext_android_cb, ext_safari)
```

Comment:
scriptlets work in Safari
`!+ PLATFORM(ios, ext_android_cb)`

Reply:
How?